hieradata/role/fiberspec.yaml

```diff
@@ -4,4 +4,9 @@ classes:
   - "profile::core::docker"
   - "profile::core::fiberspec"
   - "profile::core::nfsclient"
+  - "profile::core::systemd"
   - "profile::ts::dco"
+
+systemd::manage_udevd: true
+systemd::udev_log: 'debug'
```

Comment:
Does this need to be merged into the production config?  I'm guessing that having the host on a branch for a few hours or days would be sufficient.

Reply:
This was a request from Michael to debug issues on his nodes particularly  udev rule logs related messages. I would appreciate having this merged now, and I can negotiate with him to have it removed once he is satisfied with his tests, so I don't have the hosts sitting on a branch at the moment.